src/phoenix/core/traces.py

```diff
@@ -319,6 +324,12 @@ def _process_span(self, span: pb.Span) -> None:
         if existing_span:
             self._token_count_total -= existing_span[LLM_TOKEN_COUNT_TOTAL] or 0
         self._token_count_total += new_span[LLM_TOKEN_COUNT_TOTAL] or 0
+        # Update number of documents
+        num_documents = len(span.retrieval.documents)
+        if existing_span:
+            num_documents -= len(existing_span.retrieval.documents)
+        if num_documents:
+            self._num_documents[span_id] += num_documents
```

Comment:
Can you help understand this update logic?

Reply:
it's just the difference, but i was too lazy to create a new variable. my bad